application/dash_application/utility/plot_builder.py

```diff
@@ -559,16 +573,20 @@ def _define_graph(data, x_fn, y_fn, bar_positive, bar_negative, hovertext_cols,
     else:
         error_y = None
         hovertext_display_cols = hovertext_cols
-
     hovertext = create_data_label(data, hovertext_display_cols, additional_hovertext)
-
+    
+    additional_str = ""
+    if isinstance(hovertext, pandas.Series) and not hovertext.empty:
```

Comment:
I think any Python sequence will work here  (will silently throw away `hovertext` if user supplied it in Python `list` format), `if len(hovertext) > 0` should work.

Reply:
Fixed